qiskit/tools/qi/pauli.py

```diff
@@ -60,6 +60,23 @@ def __str__(self):
             stemp += str(j) + '\t'
         return stemp
 
+    def __eq__(self, other):
+        """Return True if all Pauli terms are equal."""
+        bres = False
+        if self.numberofqubits == other.numberofqubits:
+            if self.v == other.v and self.w == other.w:
+                    bres = True
+        return bres
+
+    def __ne__(self, other):
```

Comment:
Is defining `__ne__` really needed? By default, Python 3 will invert the result of `__eq__` - could you check and remove `__ne__` if that is the case?